iaso/api/mobile/entity.py

```diff
@@ -117,20 +83,32 @@ class MobileEntityViewSet(ModelViewSet):
     remove_results_key_if_paginated = True
     filter_backends = [filters.OrderingFilter, DjangoFilterBackend, DeletionFilterBackend]
     pagination_class = LargeResultsSetPagination
+    permission_classes = [permissions.IsAuthenticated, HasPermission("menupermissions.iaso_entities")]
     lookup_field = "uuid"
 
     def get_serializer_class(self):
         return MobileEntitySerializer
 
     def get_queryset(self):
-        profile = self.request.user.iaso_profile
+        user = self.request.user
+        app_id = self.request.query_params.get("app_id")
```

Comment:
What if we don't pass an `app_id`? I think it should be an error.

Also, you should use the constant rather than hardcoding the parameter